employee-onboarding-A2A-reference-implementation/agents/it_agent/__main__.py

```
@@ -0,0 +1,156 @@
+    async def dispatch(self, request: Request, call_next):
+        auth_header = request.headers.get('Authorization', '')
+        if auth_header.startswith('Bearer '):
+            self.executor.set_auth_token(auth_header[7:])
+        else:
+            self.executor.set_auth_token(None)
+        return await call_next(request)
+
+
+def main():
+    logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+    logger = logging.getLogger(__name__)
```

Ruff (0.15.2)

[error] Local variable `logger` is assigned to but never used

Remove assignment to unused variable `logger`

(F841)